target/linux/ipq40xx/image/generic.mk

```diff
@@ -636,6 +636,19 @@ define Device/netgear_ex6150v2
 endef
 TARGET_DEVICES += netgear_ex6150v2
 
+define Device/netgear_wac510
+	$(call Device/FitImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Netgear
+	DEVICE_MODEL := WAC510
+	SOC := qcom-ipq4018
+	DEVICE_DTS_CONFIG := config@5
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_PACKAGES := uboot-envtools ipq-wifi-netgear_wac510
```

Comment:
I think the WAC510 has the boardfiles already upstream? If it's the same version, then you can drop it.

Reply:
It was added:

[https://github.com/kvalo/ath10k-firmware/commit/d13a7c4722e97ad60b66d23033266fe908dab6cf#diff-2a6eafeadd2ed8ed8dbce8fbe39222798a0c7e312dedf89b3b361c9bedd2c7bb](url)

Reply:
yes, that's why I ask. You don't need to provide a override board-2.bin (ipq-wifi-netgear_wac510 package) if the boardfile is already integrated.

Reply:
Built and tested an image using the intergrated upstream board-2.bin without issues.

Reply:
Makes sense. Thanks!